app/models/spree/return_authorization.rb

```diff
@@ -8,8 +8,8 @@ class ReturnAuthorization < ApplicationRecord
 
     belongs_to :order, class_name: 'Spree::Order', inverse_of: :return_authorizations
 
-    has_many :inventory_units, inverse_of: :return_authorization
-    has_one :stock_location
+    has_many :inventory_units, inverse_of: :return_authorization, dependent: :nullify
+    has_one :stock_location, dependent: :nullify
```

Comment:
Not sure if this has any impact, probably not.

Reply:
keeping